### PR TITLE
Minor text edits

### PR DESCRIPTION
**Pull request by beklapko**
- Just a few minor text edits
- Definitely feel free to ignore or edit the suggestions

### CODE REVIEW

File: src/commands/signup.ts [hunk 1]
```diff
@@ -133,7 +133,7 @@ async function collectName(): Promise<string | undefined> {
   const { name } = await inquirer.prompt([
     {
       name: "name",
-      message: "What is your full name?",
+      message: "What is your first and last name?",
```
Comment (beklapko) [inline src/commands/signup.ts:136]: I assumed this was first and last name but wanted to drop a comment here to make sure.

File: src/utils/credentials.ts [hunk 2]
```diff
@@ -23,24 +23,24 @@ export function askForCookies() {
...
         name: "accessToken",
-        message: `Last thing! What is your access token? It's a very long string, also found in the cookies inspector.`,
+        message: `What is your access token? It's a very long string, also found in the cookies inspector.`,
         type: "input",
       },
     ])
     .then(function (answer: Credentials) {
       persistCredentials(answer);
-      console.log("Credentials saved/updated successfully!");
+      console.log("Credentials saved and updated successfully!");
```
Comment (beklapko) [inline src/utils/credentials.ts:43]: If "saved" and "updated" are interchangeable we can just pick one and go with it. I wasn't sure so I included both, but if they mean the same thing we can go with just one.

Comment (PeteTheHeat) [reply src/utils/credentials.ts:43]: Originally there was a difference between save and update, but I don't think it's worth highlighting to user. 

We should just go with "saved" everywhere.

File: src/commands/login.ts [hunk 3]
```diff
@@ -172,7 +172,7 @@ async function loginViaBrowser() {
       accessToken,
       xsrf: xsrfToken,
     });
-    console.log("Credentials saved/updated successfully!");
+    console.log("Credentials saved and updated successfully!");
```
Comment (PeteTheHeat) [inline src/commands/login.ts:175]: Let's go with saved like below.

**Review by PeteTheHeat — APPROVED** [review]
Thanks for this!

**Comment by PeteTheHeat** [comment]
Merged this PR and I'll follow up with the "Saved" edits, I'm about to polish the `whoami` command and don't want you to hit merge conflicts,

**Pull request merged**